homeassistant/components/here_travel_time/config_flow.py

```diff
@@ -150,37 +164,49 @@ async def async_step_origin_coordinates(
             self._config[CONF_ORIGIN_LONGITUDE] = user_input[CONF_ORIGIN][
                 CONF_LONGITUDE
             ]
+            self._config.pop(CONF_ORIGIN_ENTITY_ID, None)
```

Comment:
Add a comment explaining we pop this to clear previous configuration using an entity